platforms/cortex/platform.c

```diff
@@ -265,18 +266,62 @@ static void *pf_task_countdown(void *arg)
     return NULL;
 }
 
+static void *_task_planner_start_cancel(void *arg)
+{
+    bool *start_shell = (bool *)arg;
+
+    /* Wait for Enter to be pressed */
+    getchar();
+
+    /* Set a flag and return once done */
+    *start_shell = false;
+
+    puts("Key pressed, do not start planner.");
+
+    return NULL;
+}
+
 void pf_init_tasks(void)
 {
 
-    ctrl_t* controller = (ctrl_t*)&ctrl_quadpid;
+    ctrl_t *controller = pf_get_ctrl();
+    bool start_planner = true;
+    int countdown = PF_START_COUNTDOWN;
+
+    /* Create thread that up a flag on key pressed to not start the planner automatically */
+    kernel_pid_t planner_start_cancel_pid = thread_create(
+        planner_start_cancel_thread_stack,
+        sizeof(planner_start_cancel_thread_stack),
+        THREAD_PRIORITY_MAIN + 1, 0,
+        _task_planner_start_cancel,
+        &start_planner,
+        "wait planner start cancel"
+    );
+
+    puts("Press Enter to cancel planner start...");
+
+    /* Wait for Enter key pressed or countdown */
+    while ((start_planner) && (countdown > 0)) {
+        xtimer_ticks32_t loop_start_time = xtimer_now();
+        printf("%d seconds left...\n", countdown);
+        countdown--;
+        xtimer_periodic_wakeup(&loop_start_time, US_PER_SEC);
+    }
+
+    /* Stop task_planner_start_cancel thread if still running */
+    thread_t *planner_start_cancel_thread = (thread_t *)thread_get(planner_start_cancel_pid);
+    if (planner_start_cancel_thread) {
+        sched_set_status(planner_start_cancel_thread, STATUS_STOPPED);
+    }
```

Comment:
Just a comment for later but I think this should be placed in a dedicated module.

Reply:
I agree and I thought about that too, it could be generic, maybe later.